packages/server/lib/browsers/webkit.ts

```diff
@@ -18,7 +19,11 @@ export async function connectToNewSpec (browser: Browser, options, automation: A
   await wkAutomation.reset(options.url)
 }
 
-export async function open (browser: Browser, url, options: any = {}, automation: Automation): Promise<BrowserInstance> {
+export async function connectToExisting () {
```

Comment:
```suggestion
export function connectToExisting () {
```

Reply:
Or should we type connectToExisting as

```ts
connectToExisting: (browser: Browser, options: BrowserLaunchOpts, automation: Automation) => Promise<void>
```

(and not allow a `void` return type)?

Reply:
Made your suggested change in ce151923dab8d8e535bcc4a7be87f49de71db95